### PR TITLE
Show current bundle info

### PR DESCRIPTION
**Pull request by torkelrogstad**
image

### CODE REVIEW

File: lib/pages/tabs/testchain/mainchain/withdrawal_bundle_tab_page.dart [hunk 1]
```diff
@@ -170,3 +279,83 @@ class _WithdrawalViewState extends State<WithdrawalView> {
...
+  // https://github.com/LayerTwo-Labs/testchain/blob/ba78df157fcb9f85d898f65db43c2842ab9473ff/src/policy/corepolicy.h#L31-L32
+  static const int maxStandardTxWeight = 400000;
+
+  // https://github.com/LayerTwo-Labs/testchain/blob/ba78df157fcb9f85d898f65db43c2842ab9473ff/src/policy/corepolicy.h#L25
+  static const int witnessScaleFactor = 4;
+
+  // https://github.com/LayerTwo-Labs/testchain/blob/ba78df157fcb9f85d898f65db43c2842ab9473ff/src/policy/corepolicy.h#L25
+  static const double maxWeight = (maxStandardTxWeight / witnessScaleFactor) / 2;
+
+  Map<String, dynamic> get _values => {
```
Comment (octobocto) [inline lib/pages/tabs/testchain/mainchain/withdrawal_bundle_tab_page.dart:302]: Om du tar en kikk i dashboard_tab_page, kan dette gjøres generisk.


```
    return SailColumn(
      spacing: SailStyleValues.padding08,
      mainAxisAlignment: MainAxisAlignment.start,
      crossAxisAlignment: CrossAxisAlignment.start,
      children: decodedTX.keys.where((key) => key != 'walletconflicts').map((key) {
        return SingleValueContainer(
          label: key,
          value: decodedTX[key],
          width: width,
        );
      }).toList(),
    );
```

Comment (octobocto) [reply lib/pages/tabs/testchain/mainchain/withdrawal_bundle_tab_page.dart:302]: If it is the suffixes you want, you could create a addSuffix() function that looks at the key, and appends a fitting suffix

File: packages/sail_ui/lib/widgets/containers/single_value_container.dart [hunk 2]
```diff
@@ -33,10 +33,11 @@ class SingleValueContainer extends StatelessWidget {
           const SizedBox(
             width: 13,
           ),
-        SizedBox(
-          width: width,
-          child: SailText.secondary12(label),
-        ),
+        if (label != null)
```
Comment (octobocto) [inline packages/sail_ui/lib/widgets/containers/single_value_container.dart:36]: Do we really want this? Should figure out a fitting label instead imo. Can't see it used anywhere

Comment (torkelrogstad) [reply packages/sail_ui/lib/widgets/containers/single_value_container.dart:36]: I use it for the yet-to-be-bundled withdrawals

File: lib/pages/tabs/testchain/mainchain/withdrawal_bundle_tab_page.dart (hunk 1 above)
Comment (torkelrogstad) [reply lib/pages/tabs/testchain/mainchain/withdrawal_bundle_tab_page.dart:302]: I'm just following the design spec 🫡

**Pull request merged**